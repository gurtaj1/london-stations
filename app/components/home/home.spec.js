import Home from 'components/home/home';

describe('LS.Components', () => {
  describe('Home Container', () => {
    it('should export something', () => {
      expect(Home).toBeDefined();
    });

    it('should render without crashing', () => {
      expect(Home).toMatchSnapshot();
    });
  });
});
