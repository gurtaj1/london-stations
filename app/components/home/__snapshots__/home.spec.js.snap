// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LS.Components Home Container should render without crashing 1`] = `Object {}`;
